src/components/AbsenceList/test.tsx

```diff
@@ -3,58 +3,96 @@ import "@testing-library/jest-dom";
 import AbsenceList from ".";
 import { QueryClient, QueryClientProvider } from "@tanstack/react-query";
 import { fetchAbsences } from "../../services/api/apiCalls";
+import userEvent from "@testing-library/user-event";
+import { sortAbsences } from "../../utils/Sorting/sortAbsences";
 
 jest.mock("../../services/api/apiCalls");
+
 afterEach(() => {
-  queryClient.clear();
+  mockQueryClient.clear();
 });
 
-const queryClient = new QueryClient();
-const mockAbsenceData = [
-  {
-    id: 0,
-    startDate: "2022-05-28T04:39:06.470Z",
-    days: 9,
-    absenceType: "SICKNESS",
-    employee: {
-      firstName: "Rahaf",
-      lastName: "Deckard",
-      id: "2ea05a52-4e31-450d-bbc4-5a6c73167d17",
+const mockQueryClient = new QueryClient();
+const mockAbsenceData = sortAbsences(
+  [
+    {
+      id: 0,
+      startDate: new Date(
+        "Sat May 28 2022 05:39:06 GMT+0100 (British Summer Time)"
+      ),
+      endDate: new Date(
+        "Mon Jun 06 2022 05:39:06 GMT+0100 (British Summer Time)"
+      ),
+      days: 9,
+      absenceType: "SICKNESS",
+      employee: {
+        firstName: "Rahaf",
+        lastName: "Deckard",
+        id: "2ea05a52-4e31-450d-bbc4-5a6c73167d17",
+      },
+      approved: true,
+      conflicts: false,
+    },
+    {
+      id: 1,
+      startDate: new Date(
+        "Tue Feb 08 2022 08:02:47 GMT+0000 (Greenwich Mean Time)"
+      ),
+      endDate: new Date(
+        "Sun Feb 13 2022 08:02:47 GMT+0000 (Greenwich Mean Time)"
+      ),
+      days: 5,
+      absenceType: "ANNUAL_LEAVE",
+      employee: {
+        firstName: "Enya",
+        lastName: "Behm",
+        id: "84502153-69e6-4561-b2de-8f21f97530d3",
+      },
+      approved: false,
+      conflicts: false,
     },
-    approved: true,
-  },
-  {
-    id: 1,
-    startDate: "2022-02-08T08:02:47.543Z",
-    days: 5,
-    absenceType: "ANNUAL_LEAVE",
-    employee: {
-      firstName: "Enya",
-      lastName: "Behm",
-      id: "84502153-69e6-4561-b2de-8f21f97530d3",
+    {
+      id: 10,
+      startDate: new Date(
+        "Sun Apr 19 2022 10:17:57 GMT+0000 (Greenwich Mean Time"
+      ),
+      endDate: new Date(
+        "Sat May 2 2022 15:56:10 GMT+0000 (Greenwich Mean Time"
+      ),
+      days: 13,
+      absenceType: "ANNUAL_LEAVE",
+      employee: {
+        firstName: "Raniya",
+        lastName: "Otte",
+        id: "e10058e4-3383-466b-91d8-1ea5bf1acf0f",
+      },
+      approved: false,
+      conflicts: true,
     },
-    approved: true,
-  },
-];
+  ],
+  "startDate",
+  "DESC"
+);
 
 describe("AbsenceList", () => {
   test("renders list of absences", async () => {
     (fetchAbsences as jest.Mock).mockReturnValue(mockAbsenceData);
+
     render(
```

Comment:
you could create a renderAbsencesList function which is called in each test, would help reducing the file by a few lines